couchcopy

```diff
@@ -302,7 +330,7 @@ async def unbrand(old_archive, new_archive, tmp_dir=None):
     node_name, origin_tmp_dir = await load(
         old_archive, tmp_dir=tmp_dir, blocking=False)
     await backup('localhost', origin_tmp_dir.name + '/data', new_archive,
-                 tmp_dir=tmp_dir, nodes_names=[node_name])
```

Comment:
This change on `couchcopy backup` also affects `couchcopy unbrand`.

Reply:
> I used the --tmp-dir option a lot, because /tmp is on RAM and that limits the size of archives I can unbrand locally on my PC, or the size of CouchDB I can backup on my PC.

Me too!
> Can we keep this option when we use couchcopy locally?

There seems to be a misunderstanding (sorry if it's on my side, tell me!)
- The option is still present on the *load* part of `couchcopy unbrand` (the lines just above).
- On the *save* part, no more temporary data is written on disk, only the final destination file, which is customizable with `couchcopy backup ... /wherever/you/want/backup.tar.gz`.

Reply:
Ah!

Perfect then, it's me I didn't read carefully enough. Thanks for the answer.